api.md

```diff
@@ -1070,6 +1167,64 @@ to the server's timezone setting.
  multi-day calls to time_bucket. The old behavior can be reproduced by passing
  2000-01-01 as the origin parameter to time_bucket.
 
+## time_bucket_gapfill() [](time_bucket_gapfill)
+
+The `time_bucket_gapfill` function works similar to `time_bucket` but also activates gap
+filling for the interval between `start` and `end`. It can only be used with an aggregation
+query.
+
+#### Required Arguments [](time_bucket_gapfill-required-arguments)
+
+|Name|Description|
+|---|---|
+| `bucket_width` | A PostgreSQL time interval for how long each bucket is (interval) |
+| `time` | The timestamp to bucket (timestamp/timestamptz/date)|
+| `start` | The start of the gapfill period (timestamp/timestamptz/date)|
+| `end` | The end of the gapfill period (timestamp/timestamptz/date)|
+
+#### Required Arguments [](time_bucket_gapfill-integer-required-arguments)
+
+|Name|Description|
+|---|---|
+| `bucket_width` | integer interval for how long each bucket is (int2/int4/int8) |
+| `time` | The timestamp to bucket (int2/int4/int8)|
+| `start` | The start of the gapfill period (int2/int4/int8)|
+| `end` | The end of the gapfill period (int2/int4/int8)|
+
+#### Sample Usage [](time_bucket_gapfill-examples)
+
+Simple gap fill query with 5 minute interval:
+
+```sql
+SELECT
+  time_bucket_gapfill('5 minutes', time, now() - interval '1 week', now()) AS time,
+  avg(value)
+FROM metrics
+GROUP BY 1
+ORDER BY 1;
+```
+
+Simple gap fill query with 5 minute interval with LOCF:
```

Comment:
Could we do `LOCF (last object carried forward)` Some people don't understand LOCF for some reason.

Reply:
Also, same comment re matching your sentence description for LOCF